code/modules/organs/organ_external.dm

```diff
@@ -961,6 +961,18 @@ Note that amputating the affected organ does in fact remove the infection from t
 	species = new S.type
 	owner.update_body()
 
+/datum/organ/external/proc/plasmify()
+	if(istype(species, /datum/species/plasmaman))
```

Comment:
the macros don't work in this case, I really shouldn't be using isvox()

isvox, or isplasmaman, checks if the current type is /mob/living/carbon/human, which would not be true for an external organ datum.